x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/search/SearchApplicationIndexService.java

```diff
@@ -166,6 +166,20 @@ private static XContentBuilder getIndexMappings() {
                     builder.field("type", "keyword");
                     builder.endObject();
 
+                    builder.startObject(SearchApplication.TEMPLATE_FIELD.getPreferredName());
```

Comment:
We don't need to map these fields. The template is serialised in the binary field. The idea is that we don't want to change the mapping of the system index every time we add a section or a field in SearchApplication.